packages/block-library/src/post-date/edit.js

```diff
@@ -77,6 +78,7 @@ export default function PostDateEdit( { attributes, context, setAttributes } ) {
 						<ToolbarButton
 							icon={ edit }
 							title={ __( 'Change Date' ) }
+							dateToggleButton={ toggleButtonRef }
```

Comment:
I happened to spot this as I ended up rebasing this and fixing the merge conflict before testing today. Changed it before testing.
```
							ref={ toggleButtonRef }
```